From 0cda304f06f5721100d38afe188d14629ef6d2e0 Mon Sep 17 00:00:00 2001
From: rominator1983 <13437300+rominator1983@users.noreply.github.com>
Date: Fri, 7 Oct 2022 23:33:27 +0200
Subject: [PATCH] fixing linux build

fixing parameter name of LV2URI in call to juce_add_plugin
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d2aa077..5f1015d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,7 +29,7 @@ juce_add_plugin(SmartPedal
     PLUGIN_CODE Sgp3 
     FORMATS ${JUCE_FORMATS}
     ProductName "SmartPedal"
-    LV2_URI https://github.com/GuitarML/SmartGuitarPedal
+    LV2URI https://github.com/GuitarML/SmartGuitarPedal
     ICON_BIG resources/logo.png
     MICROPHONE_PERMISSION_ENABLED TRUE
 )
